docs/user/azure/install_upi.md

```diff
@@ -229,7 +229,7 @@ done
 Create a blob storage container and upload the generated `bootstrap.ign` file:
 
 ```sh
-az storage container create --name files --account-name ${CLUSTER_NAME}sa --public-access blob
+az storage container create --name files --account-name ${CLUSTER_NAME}sa --account-key $ACCOUNT_KEY
```

Comment:
The same comments as the above.

Reply:
As stated here, https://bugzilla.redhat.com/show_bug.cgi?id=2020216#c8 we cannot remove `--public-access blob` (without more work).